index.html

```diff
@@ -11,21 +11,50 @@
       <img src="{{image_url}}">
       <h1>{{title}}</h1>
       <h2>{{author}}</h2>
-      
     </header>
-  
-  
+  </script>
+  <script id="book-detail-template" type="text/x-handlebars-template">
+    <img src="{{image_url}}">
+    <h1>{{title}}</h1>
+    <h2>{{author}}</h2>
+  </script>
+  <script id="error-message-template" type="text/x-handlebars-template">
+    <h1>{{err}}</h1>
   </script>
```

Comment:
In your handlebars script template for the list of books should contain an anchor tag with an href linking to `/books/{{book_id}}`, this is how a user can route to the detail page and trigger the functions in your route.js file.

Also, the detail template should contain the a `<p>` tag to contain the description of the book.